src/handlers/counting/repost/embed.ts

```diff
@@ -10,7 +12,9 @@ export default async function repostWithEmbed(message: Message<true>): Promise<M
         },
       ],
     });
-  } catch {
+  } catch (err) {
+    const { guildId, channelId, id: messageId, author: { id: authorId }} = message;
+    countingLogger.error(`Failed to repost message in guild (${guildId}) [EMBED]:\n${inspect({ messageId, authorId, guildId, channelId, err })}`);
```

Comment:
I'd like you to use this message instead when logging: `Failed to repost (embed) message ${message.id}, channel ${message.channel.id}, guild ${message.guild.id}, member ${message.author.id}: ${inspect(err)}` (reference below) https://github.com/countr/countr/blob/aa1399ce953d1ccd983600f18397ab048cd16a63/src/handlers/interactions/chatInputCommands.ts#L42

I see why you're deconstructing the object here but it's not really needed after this change as well